src/PGFPlotsX.jl

```diff
@@ -12,6 +12,12 @@ using DocStringExtensions
 using Parameters
 using Requires
 
+export TikzDocument, TikzPicture
+export Axis, GroupPlot, PolarAxis
+export Plot, Plot3, Expression, EmptyLine, Coordinates,
+        Table, Graphics, Legend
+export @pgf, print_tex, latexengine, latexengine!, CUSTOM_PREAMBLE, push_preamble!
```

Comment:
are you sure you want to export `print_tex`? the manual explicitly says that this is not something that is usually called from user code.

Reply:
Yeah, true but I use it quite often in e.g. notebooks to see what the output actually is instead of round-tripping to a .tex file and I don't think `print_tex` will clash with anything else.